.github/workflows/overcloud-host-image-build.yml

```diff
@@ -94,11 +94,15 @@ jobs:
         if: inputs.centos
 
       - name: Upload CentOS Stream 8 overcloud host image artifact
-        uses: actions/upload-artifact@v3
-        with:
-          name: CentOS Stream 8 overcloud host image
-          path: /opt/kayobe/images/overcloud-centos-stream-8
-          retention-days: 7
+        run: |
+          kayobe playbook run \
```

Comment:
I expect this won't work since the virtualenv and kayobe-env haven't been activated. We should probably use kayobe-automation, with the playbook-run.sh script.